### PR TITLE
Bug/car rotates before crashing

### PR DESCRIPTION
**Pull request by DominikGuzei**
This PR fixes the bug that the car rotates a little bit before crashing. As a nice side effect it simplifies the movable mixin while keeping the same functionality ;-)

### CODE REVIEW

File: app/assets/javascripts/slotcars/shared/models/car.js.coffee [hunk 1]
```diff
@@ -20,6 +20,9 @@ Shared.Car = Ember.Object.extend Shared.Movable, Shared.Drivable, Shared.Crashab
   crossedFinishLine: false
 
   drive: (shouldAccelerate) ->
+    @checkForCrash() unless @isCrashing
+    @checkForCrashEnd()
```
Comment (tombu) [inline app/assets/javascripts/slotcars/shared/models/car.js.coffee]: Shouldn´t this line be moved into the `if @isCrashing` block (3 lines below)?

File: spec/javascripts/unit/slotcars/shared/models/car_spec.js.coffee [hunk 2]
```diff
@@ -69,6 +69,26 @@ describe 'Shared.Car', ->
...
+
+      it 'should not check for crash while crashing', ->
+        @car.set 'isCrashing', true
+        sinon.stub @car, 'moveCarInCrashingDirection'
+
+        @car.drive()
+
+        (expect @car.checkForCrash).not.toHaveBeenCalled()
+
+      it 'should check for crash', ->
```
Comment (tombu) [inline spec/javascripts/unit/slotcars/shared/models/car_spec.js.coffee]: c&p alert! ;) please modify to `should check for crash end`.

**Comment by tombu** [comment]
Besides my two little issues I really appreciate those changes :wink:

**Comment by stravid** [comment]
Great!

File: spec/javascripts/unit/slotcars/shared/models/car_spec.js.coffee (hunk 2 above)
Comment (DominikGuzei) [inline spec/javascripts/unit/slotcars/shared/models/car_spec.js.coffee]: true

File: app/assets/javascripts/slotcars/shared/models/car.js.coffee (hunk 1 above)
Comment (DominikGuzei) [inline app/assets/javascripts/slotcars/shared/models/car.js.coffee]: thanks, this actually revealed a small bug in my code ;-)

**Pull request merged**